scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestControllerTest.java

```diff
@@ -114,15 +114,15 @@ public void testCreateAndGetSchedules() throws Exception {
 	@Test
 	public void testCreateSchedule_already_existing_schedule_for_appId() throws Exception {
 		String appId = TestDataSetupHelper.generateAppIds(1)[0];
-		// Create one specific date schedule for the application.
-		callCreateSchedules(appId, 1, 0);
+		// Create one specific date schedule and two recurring schedules for the application.
+		callCreateSchedules(appId, 1, 2);
 
-		// Create two specific date schedule for the same application.
-		ResultActions resultActions = callCreateSchedules(appId, 2, 0);
+		// Create two specific date schedules and one recurring schedule for the same application.
+		ResultActions resultActions = callCreateSchedules(appId, 2, 1);
```

Comment:
You are missing the point. If I create(A), and then I create(A) again, in most worlds that would fail. In a world where the second A succeeds it is either doing an update or replace. In either case, the create() becomes a misnomer and doesn't reflect what it really does. Since you chose to hid the fact that it is a PUT vs a POST, the naming vs the behavior is just a bit odd to anyone looking at the code.